### ISSUE TITLE
Start and End chromosome position

### ISSUE THREAD

**Issue by gersonjr** [AlphaGenes/AlphaImpute2#7]
Hi,

I am tying using `AlphaPeel` to impute some animals from 0 (ungenotyped) to 50K markers. To do this, I've included in my card both `startsnp` and `endsnp` options. 
Here is an example:

```
nSnp         ,       40417
InputFilePath, genoBullsImputation
pedigree, pedigreeGeno.txt
OutputFilePath, ImpResult14
nCycles, 20
runType, multi
startsnp, 23742
endsnp, 25166
```

Although it runs without any error message, the results seem very off, with imputation accuracy (correlation) being around 0.10.   
However, when running chromosome one (`startsnp, 1` , `endsnp, 2628` ), the result looks much better, with accuracy around 0,95. 

My input files look like this:
**genoBullsImputation**
```
A00471368 1 0 0 0 1 1 1 1 2 0 0 2 0 0
A00471389 1 0 0 0 0 0 1 1 2 0 0 2 0 0 
A00471406 0 2 0 1 1 1 1 1 1 2 1 2 0 0 
A00471475 2 0 1 1 1 1 0 1 2 0 0 1 1 1
```

**pedigreeGeno.txt**
```
A11548968 A09601098 A11395392
A12225412 A09687794 A11381161
A11608547 A08687697 A11608143
A11608556 A09460260 A11341519
A11383528 A09939244 A10786854
A11383532 A08275585 A10481355
```

I also used files in the `example` folder, but it also seems not to work properly. 

I'd appreciate with you have any suggestion about what can be wrong on my files?

Thank you.

**Comment by gregorgorjanc** [comment]
@gersonjr thanks for this report - I just realised this now - are we talking about AlphaPeel or AlphaImpute2 here?

**Comment by gersonjr** [comment]
Hi @gregorgorjanc, 
I was initially using `AlphaPeel`, so that's why I mentioned this software here. 
However, if `AlphaImpute2` is a better option, I am totally fine with that. The problem is that I couldn't find in the `AlphaImpute2` documentation these `startsnp` and `endsnp` options.

**Comment by gregorgorjanc** [comment]
@gersonjr I think AlphaPeel is better for what you want to do. My understanding is that both tools work best with one chromosome at a time, but clearly we need an option to work over all chromosomes - one by one of course. I have linked this issue at https://github.com/AlphaGenes/alphapeel/issues/1. I will now close the issue here.

**Comment by RosCraddock** [comment]
Being addressed in issue [#159](https://github.com/AlphaGenes/AlphaPeel/issues/159).

**Comment by XingerTang** [comment]
Please find https://github.com/AlphaGenes/AlphaPeel/issues/1#issuecomment-3617786385
